server/index.js

```diff
@@ -0,0 +1,29 @@
+// Starting up the server
+// Use the require method on express
+// Create variable to hold it
+const express = require('express');
```

Comment:
express needs to be installed as a dependency. I tried running `npm run build` and webpack gave me a bunch of erros and I believe it's just because there is no express dependency installed yet.